README.md

```diff
@@ -177,9 +177,11 @@ If the `showClearDates` prop is set to true, an `x` shows up in the input box th
   showClearDates: PropTypes.bool,
 ```
 
-If the `showCalendarIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+If the `showInputIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+Optionally, it will display a React node set to `props.customInputIcon`
 ```
-  showCalendarIcon: PropTypes.bool
+  showInputIcon: PropTypes.bool,
+  customInputIcon: PropTypes.node
```

Comment:
Possibly... Its your call. The icon is an export out of the Font Awesome SVG, under its open license. So its definitely not a custom calendar icon.

However, this is more simple if you are just going to use the basic input icon because trying to use the traditional method of adding to the HTML below and adjusting its positioning could create user interaction difficulties (i.e the icon steals the click).

And this version has a customizable capability that someone with a truly custom calendar icon could use without difficulty.